ilctest/include/ilctest/ILCTest.h

```diff
@@ -46,6 +47,15 @@ class ILCTest{
     }
 
 
+    bool isEqual (const char* v1, const char* v2)
+    {
+      return strcmp (v1, v2) == 0;
+    }
+    template <class V1, class V2 >
+    bool isEqual (const V1& v1, const V2& v2)
```

Comment:
```suggestion
    inline bool isEqual (const char* v1, const char* v2)
    {
      return strcmp (v1, v2) == 0;
    }
    template <class V1, class V2 >
    inline bool isEqual (const V1& v1, const V2& v2)
```

just to avoid potential linking problems in downstream usage

Reply:
Thanks.  But these are methods defined within a class definition, so they are already implicitly inline; the keyword would be redundant.  Including it or not is then a matter of style; since it is not present on the existing inline members, i wouldn't favor adding inline keywords here.

Reply:
Ah, I missed that bit. Then I agree, we shouldn't need the `inline`.